pfio/v2/config.py

```diff
@@ -0,0 +1,50 @@
+import configparser
+import os
+from typing import Dict, Optional
+
+
+def _default_config_file():
+    path = os.getenv('PFIO_CONFIG_PATH')
+    if path:
+        return path
+
+    basedir = os.getenv('XDG_CONFIG_HOME')
+    if not basedir:
+        basedir = os.path.join(os.getenv('HOME'), ".config")
+
+    return os.path.join(basedir, "pfio.ini")
+
+
+def _reload_config():
+    global _config
+    config = configparser.ConfigParser()
+    configfile = _default_config_file()
+    config.read(configfile)
+    _config = config
+
+
+def add_custom_scheme(
+    name: str,
+    scheme: str,
+    data: Optional[Dict[str, str]] = None,
+) -> None:
```

Comment:
As custom scheme is still experimental, this function should be mentioned explicitly experimental and documented somehow in docs as well.

Reply:
Added docstrings: 61dd14b94eeeaab45ee8a2beee8d2e57f6852a9a